projects/azure/README.md

```diff
@@ -136,11 +136,11 @@ team who is responsible for managing the OpenShift CI Infrastructure at [#forum-
 Optionally, you can request specific long-running tests to run that are not
 running in PRs by default.
 
-| Command | Description |
-| --- | --- |
```

Comment:
Pain of markdown tables... I feel it!

I would leave it as is, because the next we will have adjust every line just to add a new line which is longer than the existing ones.

Reply:
Auto-linter does it for you :)

Reply:
Right! Super powers! Why didn't I think about it? :)